### DISCUSSION TITLE
Looker Studio Connection?

### DISCUSSION THREAD

**Discussion by rubenatterbury** [databendlabs/databend#14324] [Q&A]
Hi everyone. 
I've been exploring the use of databend for replacing our current data warehouse... However, the major drawback is the lack of integration with Looker Studio, which we use for all our vizualisations. I saw that for Deepnote for example, you can connect to Databend via the Clickhouse connector and was wondering if it would also work for Looker Studio and if there is a connector to Looker Studio in the works? 
Regards

**Comment by Xuanwo** [comment]
Seems `Looker Studio` support both mysql and pg. Have you tried them?

**Comment by ZhiHanZ** [comment]
Could you file a feature request issue for us? @rubenatterbury , we could add native support on looker studio to roadmap
